airqo-platform/AirQo-api
# Refactoring the password reset link provided in email

- What does this PR do?
Replacing Localhost link with  the Production Link inside the email for password reset
- What JIRA task is related to this PR?
No card
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/auth-service` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run dev` to test out the server.
Afterwards, start the frontend app for analytics and "forget password". You should receive a production link for reset instead of the localhost one.
- Which end points should are ready for testing?
None
